test/color/setting_test.dart

```diff
@@ -180,6 +180,68 @@ void main() {
 
         await screenMatchesGolden(tester, 'color_setting_nostroke-example-1');
       });
+
+      processingLegacySpecTest('clear() replaces background in setup() and drawn rect', (tester) async {
+        await tester.pumpWidget(
+          Processing(
+            sketch: Sketch.simple(
+              setup: (s) {
+                s.background(color: const Color(0xFF404040));
+              },
+              draw: (s) {
+                s
+                  ..noLoop()
+                  ..fill(color: const Color(0xFF969696))
+                  ..rect(rect: Rect.fromLTWH(30, 20, 55, 55))
+                  ..clear();
+              },
+            ),
+          ),
+        );
+
+        await screenMatchesGolden(tester, 'color_setting_clear-draw-then-clear');
+      });
+
+      processingLegacySpecTest('clear() replace background in setup(), then draw rect on transparent background',
```

Comment:
You already tested the `setup()` `background()` in the test above this one, so there's no need to repeat that same test here. This test should remove the `setup` implementation, and also change the name of the test, accordingly.